### ISSUE TITLE
Yoast word count broke after update to WP 5.5

### ISSUE THREAD

**Issue by web-tailor** [Yoast/yoast-acf-analysis#271]
* [x] I've read and understood the [contribution guidelines](https://github.com/Yoast/yoast-acf-analysis/blob/develop/.github/CONTRIBUTING.md).
* [x] I've searched for any related issues and avoided creating a duplicate issue.

### Please give us a description of what happened.
After updating to WP 5.5, Yoast is not counting words anymore from ACF content.

### Technical info
* WordPress version: 5.5
* ACF Content Analysis for Yoast SEO version: 2.6
* Yoast SEO version: 14.8 Premium
* ACF type: PRO
* ACF version: 5.9.0

**Comment by rootcodex** [comment]
### Please give us a description of what happened.
Same issue happening on WordPress version 5.4.2. Yoast is not count words anymore from ACF flexible fields.

### Technical info

- WordPress version: 5.4.2
- ACF Content Analysis for Yoast SEO version: 2.6
- Yoast SEO version: 14.8 - Free version
- ACF type: PRO
- ACF version: 5.9.0

**Comment by Shaner18** [comment]
Similar issue:
Running 
Wordpress 5.5
ACF Content Analysis for Yoast SEO 2.6
Advanced Custom Fields PRO 5.9
Yoast SEO 14.8

After updating to Yoast 14.8, SEO analysis & Readability see zero content... so unable to analyze posts.

**Comment by web-tailor** [comment]
There has been a new update (2.7) on 1st of september, but this has not fixed the issue

**Comment by web-tailor** [comment]
In our functions.php we have the following scripts:
```
// 1.3.2 Fully Disable Gutenberg editor.
add_filter('use_block_editor_for_post_type', '__return_false', 10);
add_action( 'wp_enqueue_scripts', 'remove_block_css', 100 );
function remove_block_css() {
wp_dequeue_style( 'wp-block-library' ); // WordPress core
wp_dequeue_style( 'wp-block-library-theme' ); // WordPress core
wp_dequeue_style( 'wc-block-style' ); // WooCommerce
wp_dequeue_style( 'storefront-gutenberg-blocks' ); // Storefront theme
}
```
```
add_action('admin_init', 'remove_standard_functions');
function remove_standard_functions() {
    remove_post_type_support( 'page', 'editor' );
    remove_post_type_support( 'page', 'comments' );
    remove_post_type_support( 'post', 'editor' );
    remove_post_type_support( 'post', 'comments' );
}
```
When I remove this, the content analysis works. But, we are not using the Gutenberg editor and leaving this there is unacceptable for our clients. Is there a way that the content analysis will work for the classic mode?

**Comment by web-tailor** [comment]
@ghindle Yes this does work! Thank you a lot. Hopefully yoast will fix this soon.

**Comment by finnruijter** [comment]
Is there a deadline for this fix? It's been 28 days and multiple clients are asking for this fix. Workaround works, but official fix would be better.

**Comment by petervinnicombe** [comment]
I can confirm this is still an issue - we are using WP5.5.1, Plugin Version 2.7 - ACF Pro 5.9.1, Yoast 14.9...
Yoast can assess everything OK such as titles / meta etc - but cannot assess anything inside the ACF fields... So word count, internal and external link, key phrase in intro, never change from red...

**Comment by davidzalar** [comment]
I'm new to Yoast and ACF contente analysis for Yoast SEO and I see there is a problem as I have the txt in widgets as the person who made the site has done it and I get word count 0. Can somebody help how to fix this isssue? Thanks in advance.
